snaps/reference/entry-points.md

```diff
@@ -214,6 +214,58 @@ module.exports.onInstall = async () => {
 </TabItem>
 </Tabs>
 
+## `onKeyringRequest`
+
+To implement the [Account Management API](keyring-api/account-management/index.md) to integrate
+[custom EVM accounts](../features/custom-evm-accounts/index.md), an account management Snap must
+expose the `onKeyringRequest` entry point.
+Whenever the Snap receives an Account Management API request, MetaMask calls the `onKeyringRequest`
+handler method.
+
+:::note
+For MetaMask to call the Snap's `onKeyringRequest` method, you must request the
+[`endowment:keyring`](permissions.md#endowmentkeyring) permission.
```

Comment:
Similar as above, could we take the user directly to the endowmentkeyring section? https://github.com/MetaMask/metamask-docs/blob/dbfb78b25696b32de5ae7613d9f7d340431701b3/snaps/reference/permissions.md#endowmentkeyring